app/controllers/UserController.js

```diff
@@ -16,13 +16,16 @@ UserController.prototype.getUsersByOrg = function () {
     var token = req.headers.authorization.split(' ')[1];
 
     User.findOne({token: token}, function (err, user) {
+      /* istanbul ignore if */
       if (err) {
         res.status(400).json(err);
+      /* istanbul ignore if */
```

Comment:
This technically doesn't even need to be here since because this is an authenticated route, meaning that the authentication system already queried the database for the user, we if we got to this point than the user exists. That is why I ignored it.  I don't think it could ever run.  With that in mind, we may want to remove it.